up

```diff
@@ -46,7 +46,6 @@ setuphosts() {
 }
 
 rundocker() {
-    # sudo -E docker-compose scale mender-client=3
     sudo -E ./demo up
```

Comment:
I disagree, your readme update states that SCALE is an env variable for `up`. `demo` is just a wrapper around docker-compose and it's only purpose is to include respective compose files. We also don't mention `demo` script in the readme because there are `up/down/update/reset`